test/object-store/sync/client_reset.cpp

```diff
@@ -5364,23 +5424,18 @@ TEST_CASE("client reset with nested collection", "[client reset][local][nested c
                     REQUIRE(nlist.get_any(8).get_int() == 7);
                 }
                 else {
+                    // db must be equal to local
```

Comment:
nit: the entire db doesn't have to be equal to the local, just anything below the list that was changed.